### PR TITLE
Improve logging of bot-ng

### PR DESCRIPTION
**Pull request by mimi1vx**
https://progress.opensuse.org/issues/109488

### CODE REVIEW

File: openqabot/loader/config.py [hunk 1]
```diff
@@ -64,16 +64,16 @@ def read_products(path: Path) -> List[Data]:
         data = loader.load(p)
 
         if not data:
-            logger.warning("something wrong with %s" % str(p))
+            logger.debug("Skiping invalid config %s" % str(p))
```
Comment (okurz) [inline openqabot/loader/config.py:67]: I suggest to not simply reuse the same text as in before but use specific, differing log messages

File: openqabot/types/incidents.py [hunk 2]
```diff
@@ -135,10 +137,11 @@ def __call__(
                             ArchVer(arch, self.settings["VERSION"])
                         ]
                     except KeyError:
-                        logger.debug(
-                            "Incident %s does not have %s arch in %s"
-                            % (inc.id, arch, self.settings["VERSION"])
-                        )
+                        if REALLY_LONG_LOG:
```
Comment (okurz) [inline openqabot/types/incidents.py:140]: What is this about?

**Review by okurz — CHANGES_REQUESTED** [review]

**Comment by okurz** [comment]
@Martchus

File: openqabot/loader/config.py (hunk 1 above)
Comment (kalikiana) [reply openqabot/loader/config.py:67]:
```suggestion
            logger.debug("Skipping invalid config %s" % str(p))
```

**Review by Martchus — COMMENTED** [review]
I only haven't added a review because I don't have anything to say in addition to @okurz 's review but all his comments should be addressed.

File: openqabot/types/incidents.py (hunk 2 above)
Comment (mimi1vx) [reply openqabot/types/incidents.py:140]: trace, usually this info has no value , only when is needed detailed info

Comment (kalikiana) [reply openqabot/types/incidents.py:140]: That's how I would describe *debug* 🤔

Comment (okurz) [reply openqabot/types/incidents.py:140]: I suggest to either just use debug or introduce a trace level like in https://www.programcreek.com/python/example/56843/logging.TRACE

File: openqabot/openqa.py [hunk 3]
```diff
@@ -20,15 +21,24 @@ def __bool__(self) -> bool:
...
+        except RequestError as e:
+            logger.error("openQA returned %s" % e[-1])
+            logger.error("Post failed with {}".format(pformat(settings)))
         except Exception as e:
             logger.exception(e)
             logger.error("Post failed with {}".format(pformat(settings)))
 
     def get_jobs(self, data: Data):
-        logger.debug("getting openqa for %s" % pformat(data))
+        logger.info("getting openqa for %s" % pformat(data))
```
Comment (baierjan) [inline openqabot/openqa.py:41]: Cosmetic suggestion (all other messages beings with a capital letter)?
```suggestion
        logger.info("Getting openQA for %s" % pformat(data))
```

**Review by baierjan — APPROVED** [review]

File: openqabot/openqa.py [hunk 4]
```diff
@@ -20,15 +21,24 @@ def __bool__(self) -> bool:
...
+        except RequestError as e:
+            logger.error("openQA returned %s" % e[-1])
+            logger.error("Post failed with {}".format(pformat(settings)))
         except Exception as e:
             logger.exception(e)
             logger.error("Post failed with {}".format(pformat(settings)))
 
     def get_jobs(self, data: Data):
-        logger.debug("getting openqa for %s" % pformat(data))
+        logger.info("Getting openqa for %s" % pformat(data))
```
Comment (Martchus) [inline openqabot/openqa.py:41]:
```suggestion
        logger.info("Getting openQA for %s" % pformat(data))
```
Same counts for other occurrences. About this specific one: Should it actually be "openQA job"?
```suggestion
        logger.info("Getting openQA job for %s" % pformat(data))
```

**Review by Martchus — APPROVED** [review]

**Pull request merged**